orc8r/cloud/go/obsidian/swagger/registry_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger_test
+
+import (
+	"testing"
+
+	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/test_utils"
+	"magma/orc8r/lib/go/registry"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRegistry_GetSpecServicers(t *testing.T) {
+	// Success with no registered servicers
```

Comment:
Test code is the correct place for these topic-sentence comments, they strike a good balance of readability vs. non-rigid test code. Thanks for adding